docs/bare_metal/mbed2_porting.md

```diff
@@ -10,7 +10,7 @@ This document describes how to configure an Mbed OS 2 target for bare metal, and
 
 The configure your target:
 
-1. Clone [mbed-os-example-blinky-baremetal](https://github.com/ARMmbed/mbed-os-example-blinky-baremetal).
+1. Import `mbed-os-example-blinky-baremetal`.
```

Comment:
Line 11 needs changing as well.

Reply:
Changed to what?

Reply:
@evedon Line11 seems to be redundant as the header also states the same. I have removed, please re-review

Reply:
It was a typo: "n" missing THe --> Then

Reply:
> It was a typo: "n" missing The --> Then

Reply:
Oh, that was supposed to be "to"! Sorry about that.